setup.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python

```

Comment:
I'm fine with this, but perhaps it should be in a separate PR since it's really separate from anything else.